docs/source/modules/introduction.rst

```diff
@@ -0,0 +1,111 @@
+
+::
+
+    RESOLUTION = T42, 25
+specifies the horizontal (in this case, spectral T42) and vertical (25 pressure levels) resolution. Runs at T42 by default. If the horizontal resolution is not T42, make sure the input files (such as land masks) are changed and add ``exp.set_resolution(*RESOLUTION)`` after exp is defined.
+
+::
+
+    cb = DryCodeBase.from_directory(GFDL_BASE)
+tells Isca to find the model code at the location ``GFDL_BASE``, which is where the source code may be changed. You can replace ``from_directory(GFDL_BASE)`` with ``from_repo(repo='https://github.com/isca/isca', commit='isca1.1')`` to point to a specific git repository and commit id. This should ensure future, independent, reproducibility of results. But, the compilation depends on computer specific settings.  The ``$GFDL_ENV`` environment variable is used to determine which ``$GFDL_BASE/src/extra/env`` file is used to load the correct compilers.  The env file is always loaded from ``$GFDL_BASE`` and not the checked out git repo.
+
+::
+
+    cb.compile()
+compiles the source code.
+
+::
+
+    exp = Experiment('EXPERIMENT_NAME', codebase=cb)
+creates an experiment object which will handle the configuration of model parameters and output diagnostics. The output files will be found at ``$GFDL_DATA/EXPERIMENT_NAME``.
+
+
+Diagnostics
+-------
+
+::
+
+    diag = DiagTable()
+creates a DiagTable object which we can configure to tell Isca which variables to output.
+
+::
+
+    diag.add_file('atmos_monthly', 30, 'days', time_units='days')
+creates an ``atmos_monthly.nc`` file which contains output from the model run every 30 days. The output files can be found at ``$GFDL_DATA/EXPERIMENT_NAME/run####/*``.
+
+::
+
+    diag.add_field(MODULE_NAME, VARIABLE_NAME, time_avg=True)
+determines which fields will be written in ``atmos_monthly.nc``. Find the available VARIABLE_NAMEs by going to the MODULE_NAME documentation or by finding the relevant source code (``cd Isca/src/ & find . -name "MODULE_NAME*"``).
+
+
+Namelist
+-------
+
+::
+
+    namelist = Namelist({...})
+defines a namelist object, which lets us configure the science options. 
+It is only necessary to set values that are different from the default parameters, which are defined 
+in the relevant module documentation (for example, ``atmosphere_nml`` parameters can be found in the ``atmosphere`` 
+module documentation or at the beginning of the ``atmosphere.F90`` source file).
+
+Running the experiment
+-------
+
+::
+
+    exp.run(...)
+will make the model run for the amount of time specified in ``main_nml`` (usually 30 days). 
+
+The ``use_restart`` option can be set to ``False`` to start from scratch (isothermal atmosphere) or can point to a restart file (``use_restart = $GFDL_DATA/exp_name/run####/restarts/*``) to initialize the run from the output of a previous run. If unspecified, it will start from where the previous run left off or from an isothermal atmosphere in the absence of a previous run.
+
+Output
```

Comment:
I think this section is nice, but it might eventually be more suited to a separate 'post-processing' section of the docs. I'd suggest leaving it here, but just noting for future.

Reply:
True, could be left here for now, but we could think of making a more elaborate post-processing doc in the future with some code templates for output analysis etc.